app/src/test/java/com/gholem/moneylab/features/chart/viewmodel/ChartViewModelTest.kt

```diff
@@ -15,22 +16,25 @@ import org.mockito.Mockito
 import org.mockito.Mockito.`when`
 import org.mockito.Mockito.verify
 
-class PlanningViewModelTest {
+class ChartViewModelTest {
     @get:Rule
     val mainCoroutineRule = MainCoroutineRule()
 
     private val insertTransactionModelUseCaseMock: InsertTransactionModelUseCase =
         Mockito.mock(InsertTransactionModelUseCase::class.java)
     private val fetchTransactionModelUseCaseMock: FetchTransactionModelUseCase =
         Mockito.mock(FetchTransactionModelUseCase::class.java)
+    private val GetTransactionListUseCaseMock: GetTransactionListUseCase =
```

Comment:
`GetTransactionListUseCaseMock` -> `getTransactionListUseCaseMock`

Also looks like there are missing unit tests for some methods as this mock is not tested anywhere